src/main/kotlin/no/nav/familie/ks/sak/kjerne/fagsak/FagsakService.kt

```diff
@@ -254,6 +258,15 @@ class FagsakService(
         }
     }
 
+    internal fun settEgenAnsattStatusPåFagsakDeltagere(fagsakDeltagere: MutableList<FagsakDeltagerResponsDto>): List<FagsakDeltagerResponsDto> {
```

Comment:
Har egentlig `internal` noe å si her? 

> internal means that any client inside this module who sees the declaring class sees its internal members.

```suggestion
    fun settEgenAnsattStatusPåFagsakDeltagere(fagsakDeltagere: MutableList<FagsakDeltagerResponsDto>): List<FagsakDeltagerResponsDto> {
```

Reply:
Nope